jobs/resource_job.go

```diff
@@ -310,8 +310,10 @@ func (js *JobSettings) sortWebhooksByID() {
 
 // JobListResponse returns a list of all jobs
 type JobListResponse struct {
-	Jobs    []Job `json:"jobs"`
-	HasMore bool  `json:"has_more,omitempty"`
+	Jobs          []Job  `json:"jobs"`
+	HasMore       bool   `json:"has_more,omitempty"`
+	NextPageToken string `json:"next_page_token,omitempty"`
+	PrevPageToken string `json:"prev_page_token,omitempty"`
```

Comment:
Just to confirm, you're adding PrevPageToken for completeness, even though we don't use it in this PR.

Reply:
Yes, only for completeness...